src/components/PaginationOld.tsx

```diff
@@ -0,0 +1,100 @@
+import { neutralColors, brandColors } from '@giveth/ui-design-system';
```

Comment:
Should we keep the other pagination component if we are not using it?

Reply:
Thank you @mateodaza  for remembering this.